### ISSUE TITLE
ETABS_Toolkit: Allow multiple versions of Adapter installed concurrently

### ISSUE THREAD

**Issue by al-fisher** [BHoM/ETABS_Toolkit#176]
Following @RichardWhitfieldTTW's good PR https://github.com/BHoM/ETABS_Toolkit/pull/175 , discussion around further extension to allow multiple versions to be complied/installed/deployed concurrently.

The use of prepossessor directives around the `using`s is a smart way of handling ETABS change in API version - without necessitating duplication of code etc.
This does not however allow two versions to be run in parallel - as the classes and dlls are the named identically. 

This is a nice to have and not critical, but we could certainly imagine further use of prepossessor directive conditions around the class name also to differentiate the version type, along with additional post-build events to preserve the parallel versions of dlls.  
Also ultimately can then give end user choice of compiled version on deployment etc. 

Notes captured here for future discussion.
@alelom @IsakNaslundBh

**Comment by RichardWhitfieldTTW** [comment]
Al, 
From ETABS18 onward the API will have the same name and be strongly typed (ie with versioning) so this should resolve this.
However the API is now targetting .Net Framework 4.7.1 which is an issue.
Rich

**Comment by IsakNaslundBh** [comment]
This can be closed as resolved with the multiple builds and similar for Etabs
